pkgs/tools/networking/wireguard-go/default.nix

```diff
@@ -2,25 +2,25 @@
   lib,
   buildGoModule,
   fetchzip,
-  testers,
+  # testers,
   wireguard-go,
 }:
 
 buildGoModule rec {
   pname = "wireguard-go";
-  version = "0.0.20230223";
+  version = "0-unstable-2023-11-10";
```

Comment:
Note: this commit has an author date of November, but was committed in December. Do we prefer a specific date?

Reply:
Probably committed date is best to go with since that's when it was actually "brought in".

Reply:
afaik commit date is what matters, but the language in the docs isn't totally clear: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md?plain=1#L391

Reply:
Yeah, could be worth some clarification. I'll switch it to commit date though (I was personally going back and forth), thanks for the insight!